packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -350,7 +350,7 @@ export default defineComponent({
       if (isPersonalSpaceResource(space)) {
         spaceBreadcrumbItem = {
           id: uuidv4(),
-          text: space.name,
+          text: $gettext('Personal'),
```

Comment:
That won't work unfortunately because this can be another user's personal space, e.g. if `fullShareOwnerPaths` is enabled. In that case the space name will be the username.

Reply:
Urgh